packages/contracts-bedrock/test/kontrol/scripts/run-kontrol.sh

```diff
@@ -135,13 +135,23 @@ regen=
 
 test_list=()
 if [ "$SCRIPT_TESTS" == true ]; then
-  test_list=( "OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused0" \
-              "OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused1(" \
-              "OptimismPortalKontrol.prove_finalizeWithdrawalTransaction_paused" \
-              "L1StandardBridgeKontrol.prove_finalizeBridgeERC20_paused" \
-              "L1StandardBridgeKontrol.prove_finalizeBridgeETH_paused" \
-              "L1ERC721BridgeKontrol.prove_finalizeBridgeERC721_paused" \
-              "L1CrossDomainMessengerKontrol.prove_relayMessage_paused"
+  test_list=(
+    "OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused0" \
```

Comment:
I think we can execute them, but atm there are two reasons why they're left out:
1. Deployment context: If I understand correctly, there are two different deployment configurations, one for each OptimismPortal version. However, both deployments are written to the `Kontrol-31337.json`, which is used to run the proofs. So, if we want to run both versions, we'll need two different JSON files to account for the different deployments.
2. CPU availability: @F-WRunTime can correct me on this, but I think our machine has 16 cores, which means that we can run at most 16 proofs in parallel. Since 12 out of the 16 proofs are OptimismPortal proofs, running the OptimismPortal2Kontrol tests here would mean running a total of 28 proofs in parallel, which I think our infrastructure doesn't support?
I might have to do some research on this

Reply:
Just updated the Temporarily unexecuted test list btw :ok_hand:

Reply:
Eventually OptimismPortal and OptimismPortal2 will be merged into one contract, but until then running both is ideal. If we need to choose just one of them for simplicity in this PR, OptimismPortal2 is preferable since that is what we currently use in production

Reply:
We have 16 cores which is 32 threads. I believe that should be fine. Do you have a rough spec of Mem requirements that may be the limiting factor @JuanCoRo

Reply:
For completeness: on my machine, with the 28 proofs running in parallel, the peak memory usage was 108GB

Reply:
For completeness:

This has been resolved by *not* including the execution of the Portal2 proofs in this PR. In a follow-up PR, we'll switch to *only* executing the Portal2 proofs instead of the current Portal proofs in a PR that will replace the `StateDiff` cheatcodes in favor of the `vm.dumpState` cheatcode